Exercise1.rb

```diff
@@ -4,3 +4,5 @@ def count_letters(s)
 puts "Enter string"
 line=gets
 print count_letters(line)
```

Comment:
Is this method counting only alphabets or everything..?
According to question it should give count of only alphabets